3p/integration.js

```diff
@@ -279,17 +279,15 @@ window.draw3p = function(opt_configCallback, opt_allowed3pTypes,
     window.context.data = data;
     window.context.noContentAvailable = triggerNoContentAvailable;
     window.context.requestResize = triggerResizeRequest;
+    window.context.renderStart = triggerRenderStart;
+    window.context.isRenderStartSent = false;
```

Comment:
This is probably not needed. But I still haven't figure out what value would Javascript script assign to a new declared value. Is it always guaranteed to be `undefined`?